File: examples/modulith-clinical-tool-jpa/src/main/java/io/zenwave360/example/clinicaltool/modules/clinical/core/inbound/dtos/PatientProfile.java
package io.zenwave360.example.clinicaltool.modules.clinical.core.inbound.dtos;

import io.zenwave360.example.clinicaltool.modules.clinical.core.domain.*;
import jakarta.validation.constraints.*;
import java.io.Serializable;
import java.math.*;
import java.time.*;
import java.util.*;

/**
 * PatientProfile.
 */
@lombok.Getter
@lombok.Setter
public class PatientProfile implements Serializable {

    @NotNull
    @Size(max = 100)
    private String name;

    @NotNull
    @Size(max = 100)
    private String surname;

    @Size(max = 100)
    private String surname2;

    @NotNull
    private LocalDate birthDate;

    @NotNull
    private GenderType gender;

    @NotNull
    @Size(max = 20)
    private String phoneNumber;
}

File: examples/modulith-klinical-tool-jpa/src/main/kotlin/io/zenwave360/example/clinicaltool/modules/clinical/core/inbound/dtos/PatientProfile.kt
package io.zenwave360.example.clinicaltool.modules.clinical.core.inbound.dtos

import io.zenwave360.example.clinicaltool.modules.clinical.core.domain.*
import jakarta.validation.constraints.*
import java.io.Serializable
import java.math.*
import java.time.*

/** PatientProfile. */
data class PatientProfile(
    @NotNull @Size(max = 100) val name: String? = null,
    @NotNull @Size(max = 100) val surname: String? = null,
    @Size(max = 100) val surname2: String? = null,
    @NotNull val birthDate: LocalDate? = null,
    @NotNull val gender: GenderType? = null,
    @NotNull @Size(max = 20) val phoneNumber: String? = null,
) : Serializable {}
